docs/designers-developers/developers/tutorials/block-tutorial/block-controls-toolbars-and-inspector.md

```diff
@@ -13,50 +13,45 @@ You can also customize the toolbar to include controls specific to your block ty
 {% codetabs %}
 {% ES5 %}
 ```js
-var el = wp.element.createElement,
-	Fragment = wp.element.Fragment
-	registerBlockType = wp.blocks.registerBlockType,
-	RichText = wp.editor.RichText,
-	BlockControls = wp.editor.BlockControls,
-	AlignmentToolbar = wp.editor.AlignmentToolbar;
-
-registerBlockType( 'gutenberg-boilerplate-es5/hello-world-step-04', {
-	title: 'Hello World (Step 4)',
-
-	icon: 'universal-access-alt',
-
-	category: 'layout',
-
-	attributes: {
-		content: {
-			type: 'string',
-			source: 'html',
-			selector: 'p',
+( function( blocks, editor, element ) {
+	var el = element.createElement;
+	var RichText = editor.RichText;
+	var AlignmentToolbar = editor.AlignmentToolbar;
+	var BlockControls = editor.BlockControls;
+
```

Comment:
I prefer passing in `wp` as the global, because than you can use additional parts without having to change what is being passed in, or a mistake in the order of items passed in.

```
( function( wp ) {
	var el = wp.element.createElement;
	var RichText = wp.editor.RichText;
	var AlignmentToolbar = wp.editor.AlignmentToolbar;
	var BlockControls = wp.editor.BlockControls;
```